src/coreclr/src/inc/utilcode.h

```diff
@@ -748,12 +748,16 @@ class CCompRC
 private:
     HRESULT GetLibrary(LocaleID langId, HRESOURCEDLL* phInst);
```

Comment:
Can these 3 methods (GetLibrary, LoadLibraryThrows, LoadLibrary) be also under `#ifdef HOST_WINDOWS` ?

Reply:
Yup, that connects it all the way up to start of the chain in `LoadString` under preexisting ifdef.